roles/network_plugin/calico/tasks/main.yml

```diff
@@ -52,6 +42,16 @@
   changed_when: false
   when: "{{ use_hyperkube_cni|bool }}"
```

Comment:
This won't be idempotent. Let's track hyperkube changing. It's a bit tricky here. We can track it from downloads or simply register previous hyperkube version in this task and see if the cmd output changed since last run.